docs/components/tabs.md

```diff
@@ -0,0 +1,348 @@
+<!--docs:
+title: "Tabs"
+layout: detail
+section: components
+excerpt: "Tabs organize and allow navigation between groups of content that are related and at the same level of hierarchy."
+iconId:
+path: /catalog/tabs
+-->
+
+# Tabs
+
+[Tabs](https://material.io/components/tabs/) organize content across different screens, data sets, and other interactions.
+
+There are two types of tabs:
+1. [Fixed tabs](#fixed-tabs)
+1. [Scrollable tabs](#scrollable-tabs)
+
+<br>
+
+1. ![fixed tab example](assets/tabs/tabs-fixed-hero.png)
+1. ![scrollabe tab example](assets/tabs/tabs-scrollable-hero.png)
+
+## Using tabs
+
+Tabs organize and allow navigation between groups of content that are related and at the same level of hierarchy.
+
+### Installing 
+To use Tabs in Flutter you need to import the Material package \
+`import 'package:flutter/material.dart';`
+
+### Making tabs accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
```

Comment:
Maybe use this same text for data tables a11y, since it's generic

Reply:
Done